Source/Core/ScreenSpaceEventHandler.js

```diff
@@ -569,14 +569,14 @@ define([
 
                 var dX = position2.x - position1.x;
                 var dY = position2.y - position1.y;
-                var dist = Math.sqrt(dX * dX + dY * dY) * 0.25;
+                var dist = Math.sqrt(dX * dX + dY * dY);
```

Comment:
@puckey It looks like you accidentally added your changes from your other PR to this one.  Can you revert them?